src/presentation/wallet/send/address-amount.tsx

```diff
@@ -33,7 +34,7 @@ const AddressAmountView: React.FC<AddressAmountProps> = ({
 
   const handleBackBtn = async () => {
     await dispatch(flushPendingTx()).catch(console.error);
-    history.goBack();
+    history.push(DEFAULT_ROUTE);
```

Comment:
replace? otherwise will add more screen on the stack

Reply:
resolved by [ba968e7](https://github.com/vulpemventures/marina/pull/380/commits/ba968e7c5069842b8f82a96921945aa6bdf3d7d1)